docker/web/nginx.conf

```diff
@@ -55,8 +55,7 @@ http {
         try_files $uri $uri/ =404;
 
         # https://www.nginx.com/resources/wiki/start/topics/examples/phpfcgi/
-        # ~ [^/]\.php(/|$)
-        location ~ \.php$ {
+        location ~ [^/]\.php(/|$) {
```

Comment:
Dunno man, we don't do the `index.php/foo/bar` thing here

Reply:
I thought it would apply to parameters as well but looking at the regex it indeed does not. Fixed in eec842a